samples/layouts/dock-manager/add-content-runtime/src/DockManagerStyles.css

```
@@ -0,0 +1,41 @@
+
+.dockManagerContent {
```

In order to easily apply the style to the newly added panes we could change it like this and remove the dockManagerContent class from the html.
```suggestion
igc-dockmanager > div {
```